src/heroes/bastion/reconfigure.opy

```diff
@@ -1,16 +1,25 @@
 #!mainFile "../../dev_main.opy"
 
+playervar sentry_healing
+
 
 rule "[bastion/reconfigure.opy]: Reduce assault form damage and movement speed":
     @Event eachPlayer
     @Hero bastion
     @Condition eventPlayer.isUsingAbility1()
     
     eventPlayer.setDamageReceived(percent(ADJ_BASTION_IRONCLAD_RESISTANCE/OW2_BASTION_IRONCLAD_RESISTANCE))
+    eventPlayer.movement_helper = percent(ADJ_BASTION_RECONFIGURE_MOVESPEED/OW2_BASTION_RECONFIGURE_MOVESPEED)
     eventPlayer.setMoveSpeed(percent(ADJ_BASTION_RECONFIGURE_MOVESPEED/OW2_BASTION_RECONFIGURE_MOVESPEED))
```

Comment:
Create a macro for `setBaseMoveSpeed()` in `utilities/macro_functions.opy` for readability/maintainability
also the name `movement_helper` is ambiguous
```suggestion
    eventPlayer.movement_helper = percent(ADJ_BASTION_RECONFIGURE_MOVESPEED/OW2_BASTION_RECONFIGURE_MOVESPEED)
    eventPlayer.setMoveSpeed(percent(ADJ_BASTION_RECONFIGURE_MOVESPEED/OW2_BASTION_RECONFIGURE_MOVESPEED))
```